app/views/welcome/index.html.erb

```diff
@@ -1,45 +1,67 @@
-<div class="jumbotron jumbotron-fluid">
-    <div class="container">
-        <div class="row">
-            <div class="col-md-8">
-                <h1 class="display-3"><span class="fa fa-handshake-o"></span>HelpOut</h1>
-                <h1 class="lead">A place to get support, and provide it.</h1>
-            </div>
-            <div class="col-md-4">
-                <h2>Sign up</h2>
+<% if current_user %>
+    <% if current_user.instructor = false %>
```

Comment:
Remember, you can use shorthand here `if !current_user.instructor` or even better `unless current_user.instructor`